### ISSUE TITLE
support google jib in future

### ISSUE THREAD

**Issue by mmpei** [goharbor/harbor#5851]
jib is a convenient CI tool and becomes more and more populcate.
and pushing images from jib could be supported by harbor.
now harbor will filter the http agent with 'docker' and 'vic' when receving a OnPush notification.
jib agent is like follow:
**http.request.useragent="jib 0.9.10 jib-maven-plugin Google-HTTP-Java-Client/1.23.0 (gzip)"**

should add jib when filter in file github.com\vmware\harbor\src\ui\service\notifications\registry\handler.go
```
//pull and push manifest by docker-client or vic
		if (strings.HasPrefix(event.Request.UserAgent, "docker") || strings.HasPrefix(event.Request.UserAgent, vicPrefix) || strings.HasPrefix(event.Request.UserAgent, "jib")) &&
			(event.Action == "pull" || event.Action == "push") {
			events = append(events, &event)
			log.Debugf("add event to collect: %s", event.ID)
			continue
		}
```

**Comment by steven-zou** [comment]
@AgoodMITer 

Thanks for raising proposal to us. I'm wondering are you willing to resolve it and contribute code back to harbor community? We're encouraging community contributors to participate more in the harbor community.

**Comment by reasonerjt** [comment]
Fixed in #5873
